### PR TITLE
fix!: update x/collection,token proto

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

It would apply the client's requirements on x/collection,token proto.
- Update the class-related part in the proto as requested by our client.
- Remove proxy in the proto.
- Replace all the uri variants into uri, in the proto.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by ByunghyunBang** [comment]
Could check comment?
Fix `approver` -> `holder`, `proxy` -> `operator`
* token/v1/tx.proto
- rpc AuthorizeOperator
- rpc OperatorSend
- rpc OperatorBurn
- rpc Modify
- message MsgOperatorSend
- message MsgAuthorizeOperator
- message MsgOperatorBurn
* collection /v1/tx.proto
- rpc OperatorSendFT
- rpc OperatorSendNFT
- rpc AuthorizeOperator
- rpc RevokeOperator
- rpc OperatorBurnFT
- rpc OperatorBurnNFT
- rpc Modify
- rpc OperatorAttach
- rpc OperatorDetach

**Comment by 0Tech** [comment]
> Could check comment?
> Fix `approver` -> `holder`, `proxy` -> `operator`

The conditions on the errors would be wiped out from the comment, because we will introduce its own codespace. It would be obvious from the error code.

**Comment by ByunghyunBang** [comment]
Could you check these? `approver` and `proxy` are used here

## collection/v1/query.proto
rpc Approved(QueryApprovedRequest) returns (QueryApprovedResponse)
rpc Approvers(QueryApproversRequest) returns (QueryApproversResponse)
message QueryApprovedRequest
message QueryApprovedResponse
message QueryApproversRequest
message QueryApproversResponse

## token/v1/query.proto
rpc Approved(QueryApprovedRequest) returns (QueryApprovedResponse)
rpc Approvers(QueryApproversRequest) returns (QueryApproversResponse)
message QueryApprovedRequest
message QueryApprovedResponse
message QueryApproversRequest
message QueryApproversResponse

example)
```
// QueryApprovedRequest is the request type for the Query/Approved RPC method
message QueryApprovedRequest {
  // contract id associated with the contract.
  string contract_id = 1;
  // address of the proxy which the authorization is granted to.
  string proxy = 2;
  // approver is the address of the approver of the authorization.
  string approver = 3;
}

// QueryApprovedResponse is the response type for the Query/Approved RPC method
message QueryApprovedResponse {
  bool approved = 1;
}
```

File: proto/lbm/collection/v1/event.proto [hunk 1]
```diff
@@ -138,8 +138,8 @@ message EventCreatedFTClass {
   string contract_id = 1;
   // address which triggered the create.
   string operator = 2;
-  // class id associated with the token class.
-  string class_id = 3;
+  // token id associated with the token class.
+  string token_id = 3;
```
[the remark this answers is not in the document]
Comment (0Tech) [reply proto/lbm/collection/v1/event.proto:142]: Yes, the change has been applied.

File: proto/lbm/collection/v1/event.proto [hunk 2]
```diff
@@ -256,8 +256,8 @@ message EventModifiedTokenClass {
   string contract_id = 1;
   // address which triggered the modify.
   string operator = 2;
-  // class id associated with the token class.
-  string class_id = 3;
+  // token type associated with the token class.
+  string token_type = 3;
   // changes of the attributes applied.
   repeated Attribute changes = 4 [(gogoproto.nullable) = false];
```
[the remark this answers is not in the document]
Comment (0Tech) [reply proto/lbm/collection/v1/event.proto]: I added the comments.

File: x/token/client/cli/tx.go [hunk 3]
```diff
@@ -89,7 +89,7 @@ func NewTxCmdSend() *cobra.Command {
 	return cmd
 }
 
-func NewTxCmdTransferFrom() *cobra.Command {
+func NewTxCmdOperatorSend() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "transfer-from [contract-id] [operator] [from] [to] [amount]",
```
Comment (jaeseung-bae) [inline x/token/client/cli/tx.go:94]: cli `Use` string may need updates("transfer-from " -> "operator-send ")

File: x/token/client/cli/tx.go [hunk 4]
```diff
@@ -391,7 +391,7 @@ func NewTxCmdBurn() *cobra.Command {
 	return cmd
 }
 
-func NewTxCmdBurnFrom() *cobra.Command {
+func NewTxCmdOperatorBurn() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "burn-from [contract-id] [grantee] [from] [amount]",
```
Comment (jaeseung-bae) [inline x/token/client/cli/tx.go:396]: cli `Use` string may need updates("burn-from " -> "operator-burn ")

File: x/collection/client/cli/tx.go [hunk 5]
```diff
@@ -44,29 +44,29 @@ func NewTxCmd() *cobra.Command {
...
+		NewTxCmdRevokeOperator(),
 	)
 
 	return txCmd
 }
 
-func NewTxCmdTransferFT() *cobra.Command {
+func NewTxCmdSendFT() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "transfer-ft [contract-id] [from] [to] [amount]",
```
Comment (jaeseung-bae) [inline x/collection/client/cli/tx.go:71]: cli `Use` string may need updates("transfer-ft xx xx" -> "send-ft xx xx")

File: x/collection/client/cli/tx.go [hunk 6]
```diff
@@ -108,7 +108,7 @@ func NewTxCmdTransferFT() *cobra.Command {
 	return cmd
 }
 
-func NewTxCmdTransferFTFrom() *cobra.Command {
+func NewTxCmdOperatorSendFT() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "transfer-ft-from [contract-id] [operator] [from] [to] [amount]",
```
Comment (jaeseung-bae) [inline x/collection/client/cli/tx.go:113]: cli `Use` string may need updates("transfer-ft-from xx xx" -> "operator-send-ft xx xx")

File: x/collection/client/cli/tx.go [hunk 7]
```diff
@@ -151,7 +151,7 @@ func NewTxCmdTransferFTFrom() *cobra.Command {
 	return cmd
 }
 
-func NewTxCmdTransferNFT() *cobra.Command {
+func NewTxCmdSendNFT() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "transfer-nft [contract-id] [from] [to] [token-id]",
```
Comment (jaeseung-bae) [inline x/collection/client/cli/tx.go:156]: cli `Use` string may need updates("transfer-nft xx xx" -> "send-nft xx xx")

File: x/collection/client/cli/tx.go [hunk 8]
```diff
@@ -187,7 +187,7 @@ func NewTxCmdTransferNFT() *cobra.Command {
 	return cmd
 }
 
-func NewTxCmdTransferNFTFrom() *cobra.Command {
+func NewTxCmdOperatorSendNFT() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "transfer-nft-from [contract-id] [operator] [from] [to] [amount]",
```
Comment (jaeseung-bae) [inline x/collection/client/cli/tx.go:192]: cli `Use` string may need updates("transfer-nft-from " -> "operator-send-nft ")

File: x/collection/client/cli/tx.go [hunk 9]
```diff
@@ -551,7 +551,7 @@ func NewTxCmdBurnFT() *cobra.Command {
 	return cmd
 }
 
-func NewTxCmdBurnFTFrom() *cobra.Command {
+func NewTxCmdOperatorBurnFT() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "burn-ft-from [contract-id] [operator] [from] [amount]",
```
Comment (jaeseung-bae) [inline x/collection/client/cli/tx.go:556]: cli `Use` string may need updates("burn-ft-from " -> "operator-burn-ft ")

File: x/collection/client/cli/tx.go [hunk 10]
```diff
@@ -628,7 +628,7 @@ func NewTxCmdBurnNFT() *cobra.Command {
 	return cmd
 }
 
-func NewTxCmdBurnNFTFrom() *cobra.Command {
+func NewTxCmdOperatorBurnNFT() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "burn-nft-from [contract-id] [operator] [from] [token-id]",
```
Comment (jaeseung-bae) [inline x/collection/client/cli/tx.go:633]: cli `Use` string may need updates("burn-nft-from " -> "operator-burn-nft ")

File: x/collection/client/cli/tx.go [hunk 11]
```diff
@@ -776,7 +776,7 @@ func NewTxCmdDetach() *cobra.Command {
 	return cmd
 }
 
-func NewTxCmdAttachFrom() *cobra.Command {
+func NewTxCmdOperatorAttach() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "attach-from [contract-id] [operator] [holder] [subject] [target]",
```
Comment (jaeseung-bae) [inline x/collection/client/cli/tx.go:781]: cli `Use` string may need updates("attach-from xx xx" -> "operator-attach xx xx")

**Review by jaeseung-bae — APPROVED** [review]

**Review by zemyblue — APPROVED** [review]

**Pull request merged**